projects/uitdatabank/docs/entry-api/project-flow.md

```diff
@@ -0,0 +1,60 @@
+---
+stoplight-id: 73ef6e15ee3ec
+---
+
+# Project flow from test to production
+
+For the integration with the UiTdatabank Entry API, there is a flow in which you first obtain access to our test environment. 
+Access to production environment is given after your integration on test environment has been successful validated by publiq vzw.
+
+On this page we describe how this flow works and how you can ensure that you gain access to the production environment as quickly as possible!
+
+> Make sure you read this guide before you start developing on our test environment!
+
+## Step 1: start on test environment
+
+You can apply for free test credentials at [Projectaanvraag](https://projectaanvraag.uitdatabank.be). 
```

Comment:
Also, on https://docs.publiq.be/docs/authentication/requesting-credentials#uitdatabank I currently have slightly different documentation on how to obtain credentials to authenticate with Entry API. We should probably align those?

Reply:
Not sure if this is needed now? We will probably "release" the docs together with publiq platform, and therefore replace "Projectaanvraag" with "publiq platform", where integrators can get an apiKey or client ID.